api/DeviceAPI.cs

```diff
@@ -21,6 +21,11 @@ public Interfaces(Uri serverAddress)
 		_http = new HttpClient();
 		_server = serverAddress;
 	}
+	public Interfaces()
+	{
+		_http = new HttpClient();
+		_server = new Uri ("http://serverapi1.azurewebsites.net");
+	}
```

Comment:
Can't you call the normal constructor with the URL you want, instead of hard-coding a value in a new constructor?